### PR TITLE
Add hook impl

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
This will be a fairly small PR, just to introduce the hook implementation.

solves: #2

### CODE REVIEW

File: everviz/main.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@hookimpl
+def visualize_data(everest_api):
+    config = webviz_config()
+
+    fname = "everviz_webviz_config.yml"
+    write_webviz_config(config, fname)
+
+    # The entry point of webviz is to call it from command line, and so we do.
+    if shutil.which("webviz"):
+        subprocess.call("webviz build {}".format(fname))
```
Comment (oyvindeide) [inline everviz/main.py]: include --theme equinor? The theme in itself is open, but only quasi open.

Comment (DanSava) [reply everviz/main.py]: I agree with adding the Equinor theme for now. We can remove it later very easily if we have problems with the "openes" of the package.

Comment (lars-petter-hauge) [reply everviz/main.py]: Added

**Comment by lars-petter-hauge** [comment]
I've added a test checking that the hook implementation is added correctly. I did not add tests for checking the functionality of the hookimplementation itself. First off, it's fairly small. And likely there will be changes to it in the upcoming PR from @oyvindeide

**Pull request merged**